tests/cli/test_cli_print_url.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import subprocess
+import unittest
+
+
+class TestPrintUrlArgument(unittest.TestCase):
+    """Test CLI for print-url argument."""
+
+    def test_print_url_argument(self):
+        url = r'https://raw.github.com/mozilla/mozdownload/master/README.md'
+        try:
+            url_arg = '--url={url}'.format(url=url)
```

Comment:
Ok, one last thing then we can land it. Please bear with me. :)

There is no need for this extra variable. Just move this line into the list below. Thanks.